### ISSUE TITLE
ReactWrapper and VueWrapper can't show the children vnode

### ISSUE THREAD

**Issue by ppfalling** [akxcv/vuera#127]
<template>
  <div>    
    <my-react :route="$route">
      **<a>!!!!!children：react children</a>     can't render **  
    </my-react>
  </div>
</template>

<script>
import { ReactWrapper } from 'vuera'
import myReact from '../myReact.tsx'
export default {
  components:{
    myReact
  }
  
}
</script>

<style>

</style>

**Comment by DaiQiangReal** [comment]
Of course. Since Vuera just using React to show your react component in vue, and do the vue props transform work for your react component. Vuera didn't transform vue format virtualDOM to React format virtualDOM. What your react componet and react get in the children props isn't a real react virtualDOM actually.

Hope there is one day that vuera can support transforming virtualDOM between react and vue. It is hard but worthly.
